locale/ru/site.json

```diff
@@ -0,0 +1,152 @@
+{
+  "title": "Node.js",
+  "author": "Node.js Foundation",
+  "url": "https://nodejs.org/en/",
+  "locale": "ru",
+  "scrollToTop": "Наверх",
+  "reportNodeIssue": "Вопрос Node.js отчет ",
+  "reportWebsiteIssue": "Вопрос сайт отчет ",
+  "getHelpIssue": "Получить Помощь",
+  "by": "по",
+  "all-downloads": "Все варианты скачать ",
+  "nightly": "Ночные сборки",
+  "chakracore-nightly": "Узел-ChakraCore ночные сборки",
+  "previous": "Предыдущие",
+  "next": "Далее",
+  "feeds": [
+    {
+      "link": "feed/blog.xml",
+      "text": "Блог Node.js "
+    },
+    {
+      "link": "feed/releases.xml",
+      "text": "Блог Node.js : Релизы"
+    },
+    {
+      "link": "feed/vulnerability.xml",
+      "text": "Блог Node.js : Сообщения Об Уязвимостях "
+    }
+  ],
+  "home": {
+    "text": "Дома"
+  },
+  "about": {
+    "link": "about",
+    "text": "О",
+    "governance": {
+      "link": "about/governance",
+      "text": "Управление"
+    },
+    "community": {
+      "link": "about/community",
+      "text": "Сообщество"
+    },
+    "workinggroups": {
+      "link": "about/working-groups",
+      "text": "Рабочие Группы"
+    },
+    "releases": {
+      "link": "about/releases",
+      "text": "Релизы"
+    },
+    "resources": {
+      "link": "about/resources",
+      "text": "Ресурсы"
+    },
+    "trademark": {
+      "link": "about/trademark",
+      "text": "Товарный знак"
+    },
+    "privacy": {
+      "link": "about/privacy",
+      "text": "Политика Конфиденциальности "
+    }
+  },
+  "download": {
+    "link": "download",
+    "text": "Загрузок",
+    "releases": {
+      "link": "download/releases",
+      "text": "Предыдущие Релизы"
+    },
+    "package-manager": {
+      "link": "download/package-manager",
+      "text": "Установка Node.js через менеджер пакетов "
+    },
+    "shasums": {
+      "link": "SHASUMS256.txt.asc",
+      "text": "Подписано SHASUMS для файлов релиз ",
+      "verify-link": "https://github.com/nodejs/node#verifying-binaries",
+      "verify-text": "How to verify"
+    },
+    "install-on-linux": {
+      "text": "Установка Node.js через бинарный архив в Linux"
+    }
+  },
+  "docs": {
+    "link": "docs",
+    "text": "Документы",
+    "es6": {
+      "link": "docs/es6",
+      "text": "ES6 и за его пределами"
+    },
+    "api-lts": {
+      "link": "/dist/latest-%ver-major%/docs/api",
+      "subtext": "LTS",
+      "text": "%Вер% по API"
```

Comment:
I guess this interpolation won't work as it's not `%ver%` exactly? For this few I would just go with `%ver% API`